src/firstscreen.h

```diff
@@ -13,6 +13,7 @@
 #include <QTextStream>
 #include <QUuid>
 #include <QSettings>
+#include <klocalizedstring.h>
```

Comment:
klocalizedstring.h is not used in firstscreen.h so it should be included in firstscreen.cpp, not here.